### PR TITLE
Add ability to replay match from specified event

### LINKED ISSUE #35: Log timetravel to quickly debug bugs
Run a log from a specific time - log will then be played at "game speed"

### PR DESCRIPTION
**Pull request by trarbr**
Closes #35

### CODE REVIEW

**Review by nicoeg — COMMENTED** [review]
This is so cool! 🤩 

Seems like the total event count is going up as the events are being played. Would it be possible for it to load them all at the beginning?
image

Also could you add a comment to the readm explaining how to start it and link to one of the log files as examples?

**Comment by nicoeg** [comment]
It could also be nice to be able to pause it but let's save that for later

File: assets/css/app.scss [hunk 1]
```diff
@@ -371,7 +371,7 @@ body {
     }
 }
 
-.match-stats {
+.match-debug {
     background: rgba(0, 0, 0, 0.3);
     color: #fff;
     padding: 20px;
```
Comment (nicoeg) [inline assets/css/app.scss:375]:
```suggestion
    padding: 20px;
    margin-top: 20px;
```

**Review by nicoeg — CHANGES_REQUESTED** [review]
I've pushed a commit that fixes the white background fade issue. Hope it's okay 😛

**Comment by trarbr** [comment]
> This is so cool! 🤩
> 
> Seems like the total event count is going up as the events are being played. Would it be possible for it to load them all at the beginning?
> 
> Also could you add a comment to the readm explaining how to start it and link to one of the log files as examples?

Great idea :+1: I have added `CsgoStats.load/1` to load all events at once, and then it lets you jump between events as you like to. I have also added a description of load and playback functions to the readme.

**Review by nicoeg — APPROVED** [review]
Cool stuff!

Just an idea: Logfile name could be set as `x-server-instance-token` so that multiple logfiles could be opened at the same time

**Pull request merged**